pkg/upgrade/upgrades/desc_id_sequence_for_system_tenant.go

```diff
@@ -27,14 +27,21 @@ func descIDSequenceForSystemTenant(
 	if !d.Codec.ForSystemTenant() {
 		return nil
 	}
-	mut := tabledesc.NewBuilder(systemschema.DescIDSequence.TableDesc()).BuildCreatedMutableTable()
+	mut := tabledesc.NewBuilder(systemschema.DescIDSequence.TableDesc()).
+		BuildCreatedMutableTable()
```

Comment:
You can get rid of this entirely and replace the sole remaining usage of `mut` with `systemschema.DescIDSequence`, since we're not messing with the start value any more.

Reply:
🤷

Reply:
Done.

Reply:
Don't get me wrong, it's so obviously inconsequential that I didn't think the comment needed to be prefixed with "nit :".